index.html

```diff
@@ -130,132 +108,90 @@ <h3 class="heading">Professional Art Printing Data</h3>
                   <div class="card-profession">
                     <h3 class="heading">Professional Art Printing Data</h3>
                     <p class="card-text-portfolio">
                       A daily selection of privately personalized reads; no accounts
                       or sign-ups required. has been the industry's standard
                     </p>
-                    <div class="skill-portfolio-3">
-                      <span class="badge"
-                        >HTML</span
-                      >
-                      <span class="badge"
-                        >Bootstrap</span
-                      >
-                      <span class="badge"
-                        >Ruby</span
-                      >
+                    <div class="skill-portfolio">
+                      <span class="badge-1">HTML</span>
+                      <span class="badge-1">Bootstrap</span>
+                      <span class="badge-1">Ruby</span>
                     </div>
                     <div class="project-see">
                       <h1 class="see-project">see project</h1>
                     </div>
                   </div>
                 </div>
-              </div>
-              </div>
-                <div class="row-portfolio">
-                   <div class="card-portfolio">
-                      <div class="card-4">
+                    <div class="card-4">
                     <div class="card-profession">
                       <h3 class="heading">Professional Art Printing Data</h3>
                       <p class="card-text-portfolio">
                         A daily selection of privately personalized reads; no accounts
                         or sign-ups required. has been the industry's standard
                       </p>
                       <div class="skill-portfolio">
-                        <span class="badge"
-                          >HTML</span
-                        >
-                        <span class="badge"
-                          >Bootstrap</span
-                        >
-                        <span class="badge"
-                          >Ruby</span
-                        >
+                        <span class="badge-1">HTML</span>
+                        <span class="badge-1">Bootstrap</span>
+                        <span class="badge-1">Ruby</span>
                       </div>
                       <div class="project-see">
                         <h1 class="see-project">see project</h1>
                       </div>
                     </div>
                   </div>
-                </div>
-                </div>
-                <div class="row-portfolio">
-                  <div class="card-portfolio">
-                    <div class="card-5">
+                  <div class="card-5">
+                    <div class="card-profession">
+                      <h3 class="heading">Professional Art Printing Data</h3>
+                      <p class="card-text-portfolio">
```

Comment:
- [ ] As the previous code reviewer mentioned, the project texts are overlapping in all the project cards in mobile and desktop versions. It is also adding a visible scroll bar for the texts. I can see you added a fixed height to the styling of the card texts which is causing the text to overlap and also added a scroll bar. Kindly fix that 😄 .

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/178066316-87b3c129-b37d-4ca5-9b30-8b6f70b1658b.png)  | ![image](https://user-images.githubusercontent.com/63473584/178066233-978cb771-5e5d-495a-900d-6ec2c88a0f68.png)  |